source/containers.tex

```diff
@@ -11237,7 +11237,8 @@
 and
 meets the requirements for
 constexpr iterators\iref{iterator.requirements.general},
-whose \tcode{value_type} is \tcode{remove_cv_t<ElementType>}.
+whose \tcode{value_type} is \tcode{remove_cv_t<ElementType>} and
+whose \tcode{reference_type} is \tcode{ElementType&}.
```

Comment:
Since `span` has nested types `value_type` and `reference`, why not simply use those? (i.e., `whose value type is \tcode{value_type} and whose reference type is \tcode{reference}`)

Reply:
Excellent!